src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int SIZE_OF_ARRAY = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[SIZE_OF_ARRAY];
+        int halfOfArray = SIZE_OF_ARRAY / 2;
```

Comment:
I'd say it's not worth to make additional variable for single usage of such simple logic, may be easily inlined